docs/Advanced-Concepts/Chip-Communication.rst

```diff
@@ -45,33 +45,21 @@ Using the Tethered Serial Interface (TSI)
-sends the TSI command recieved by the simulation stub into the DUT which then converts the TSI
-command into a TileLink request. This conversion is done by the ``SerialAdapter`` module
-(located in the ``generators/testchipip`` project). In simulation, FESVR
-resets the DUT, writes into memory the test program, and indicates to the DUT to start the program
-through an interrupt (see :ref:`customization/Boot-Process:Chipyard Boot Process`). Using TSI is currently the fastest
-mechanism to communicate with the DUT in simulation.
-
-In the case of a chip tapeout bringup, TSI commands can be sent over a custom communication
-medium to communicate with the chip. For example, some Berkeley tapeouts have a FPGA
-with a RISC-V soft-core that runs FESVR. The FESVR on the soft-core sends TSI commands
-to a TSI-to-TileLink converter living on the FPGA (i.e. ``SerialAdapter``). After the transaction is
-converted to TileLink, the ``TLSerdesser`` (located in ``generators/testchipip``) serializes the
-transaction and sends it to the chip (this ``TLSerdesser`` is sometimes also referred to as a
-serial-link or serdes). Once the serialized transaction is received on the
-chip, it is deserialized and masters a bus on the chip. The following image shows this flow:
-
-.. image:: ../_static/images/chip-bringup.png
-
-.. note::
-    The ``TLSerdesser`` can also be used as a slave (client), so it can sink memory requests from the chip
-    and connect to off-chip backing memory. Or in other words, ``TLSerdesser`` creates a bi-directional TileLink
-    interface.
+TSI protocol is an implementation of HTIF that is used to send commands to the RISC-V DUT.
+These TSI commands are simple R/W commands that are able to probe the DUT's memory space.
+During simulation, the host sends TSI commands to a simulation stub in the test harness called ``SimSerial``
+(C++ class) that resides in a ``SimSerial`` Verilog module (both are located in the ``generators/testchipip``
+project).
+This ``SimSerial`` Verilog module then sends the TSI command recieved by the simulation stub
+to an adapter that converts the TSI command into a TileLink request.
+This conversion is done by the ``SerialAdapter`` module (located in the ``generators/testchipip`` project).
+After the transaction is converted to TileLink, the ``TLSerdesser`` (located in ``generators/testchipip``) serializes the
+transaction and sends it to the chip (this ``TLSerdesser`` is sometimes also referred to as a digital serial-link or SerDes).
+Once the serialized transaction is received on the chip, it is deserialized and masters a TileLink bus on the chip
+which handles the request.
+In simulation, FESVR resets the DUT, writes into memory the test program, and indicates to the DUT to start the program
+through an interrupt (see :ref:`customization/Boot-Process:Chipyard Boot Process`).
+Using TSI is currently the fastest mechanism to communicate with the DUT in simulation and is also used by FireSim.
```

Comment:
If this is in comparison to DTM/JTAG it might be worth mentioning that? Also not sure if it is worth talking about the width here.

Reply:
I think we can glide over the width here since that is really an implementation detail that can be changed (if the user understands what TSI is). In any case we briefly discuss AXI vs TSI widths in the testchip section.